web/src/settings_linkifiers.ts

```diff
@@ -219,20 +241,21 @@ export function build_page() {
             $linkifier_status.hide();
             $pattern_status.hide();
             $template_status.hide();
-            const linkifier = {};
+            const linkifier: {[key in string]?: string} & {id?: number} = {};
```

Comment:
Although I have typed this `linkifier` object, I could understand its use. Is it redundant ?

Reply:
I don't understand it either; seems like it was replaced with the `data: $(this).serialize()` logic. @sahil839 can you confirm it looks like dead code?

Reply:
Yes we are not using the `linkifier` object.